docs/wmco-prerequisites.md

```diff
@@ -20,10 +20,8 @@ these errors, only use the appropriate version according to the cloud provider i
 
 | Cloud Provider      | Supported Windows Server version                                                     |
 | -----------         | -----------                                                                          |
-| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2022, version 21H2 |
-|                     | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019               |
-| Azure               | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2022, version 21H2 |
-|                     | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019               |
+| AWS                 | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019, version 1809 |
+| Azure               | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019, version 1809 |
```

Comment:
I'll squash these `.md` changes into the `[docs]` commit after a round of review.

Reply:
sounds good